release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1669,6 +1669,147 @@ If these values are not defined in a failure domain configuration, for instance
+
+* Previously, when installing a cluster on an {ibm-title} platform and adding an existing VPC to the cluster, the {cap-ibm-first} would not add ports 443, 5000, and 6443 to the security group of the VPC. This situation prevented the VPC from being added to the cluster. With this release, a fix ensures that the {cap-ibm-first} adds the ports to the security group of the VPC so that the VPC gets added to your cluster.(link:https://issues.redhat.com/browse/OCPBUGS-44068[*OCPBUGS-44068*])
+
+* Previously, the {cap-ibm-first} module was very verbose. With this release, the verbosity of the module is reduced, and this will affect the output of the `.openshift_install.log` file. (link:https://issues.redhat.com/browse/OCPBUGS-44022[*OCPBUGS-44022*])
+
+* Previously, when you deployed a cluster on a {cap-ibm-first} zone, the load balancers were slow to create. As a consequence, the cluster failed. With this release, the {cap-ibm-first} provider no longer has to wait until all load balancers are ready and the issue is resolved.
+(link:https://issues.redhat.com/browse/OCPBUGS-43923[*OCPBUGS-43923*])
+
+* Previously, for the Agent-based Installer, all host validation status logs referred to the name of the first registered host. As a consequence, when a host validation failed, it was not possible to determine the problem host. With this release, the correct host is identified in each log message and now the host validation logs correctly show the host to which they correspond, and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-43768[*OCPBUGS-43768*])
+
+* Previously, when you use the `oc adm node-image create` command to generate the image while running the Agent-based Installer and the step fails, the accompanying error message does not show the container’s log. The `oc adm node-image create` command uses a container to generate the image. When the image generation step fails, the basic error message does not show the underlying issue that caused the image generation failure. With this release, to help troubleshooting, the `oc adm node-image create` command now shows the container’s log, so the underlying issue is displayed and the issue is resolved.
+(link:https://issues.redhat.com/browse/OCPBUGS-43757[*OCPBUGS-43757*])
+
+* Previously, the Agent-based Installer failed to parse the `cloud_controller_manager` parameter in the `install-config.yaml` configuration file. This resulted in the Assisted Service API failing because it received an empty string, and this in turn caused the installation of the cluster to failed on {oci-first}. With this release, an update to the parsing logic ensures that the Agent-based Installer correctly interprets the `cloud_controller_manager` parameter so that the Assisted Service API receives the correct string value. As a result, the Agent-based Installer can now installer a cluster on {oci}. (link:https://issues.redhat.com/browse/OCPBUGS-43674[*OCPBUGS-43674*])
```

Comment:
```suggestion
* Previously, the Agent-based Installer failed to parse the `cloud_controller_manager` parameter in the `install-config.yaml` configuration file. This resulted in the Assisted Service API failing because it received an empty string, and this in turn caused the installation of the cluster to fail on {oci-first}. With this release, an update to the parsing logic ensures that the Agent-based Installer correctly interprets the `cloud_controller_manager` parameter so that the Assisted Service API receives the correct string value. As a result, the Agent-based Installer can now installer a cluster on {oci}. (link:https://issues.redhat.com/browse/OCPBUGS-43674[*OCPBUGS-43674*])
```